src/managed/Microsoft.NET.HostModel/Bundle/Bundler.cs

```diff
@@ -224,14 +223,13 @@ public string GenerateBundle(string sourceDir)
             // Sort the file names to keep the bundle construction deterministic.
             Array.Sort(sources, StringComparer.Ordinal);
 
-            var fileSpec = new (string SourcePath, string RelativePath)[sources.Length];
-            for(int i=0; i < sources.Length; i++)
+            List<FileSpec> fileSpecs = new List<FileSpec>();
```

Comment:
Pass sources.Length to constructor here to avoid unnecessary resizing.

Reply:
@nguerrera: done, thanks.